OPConsultation-output.json

```diff
@@ -0,0 +1,2874 @@
+          "coding": [
+            {
+              "code": "80785-9",
+              "system": "http://loinc.org",
+              "display": "Radiation oncology Plan of care note"
+            }
+          ],
+          "text": "Radiation oncology Plan of care note"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "type": "Patient"
+        },
+        "content": [
+          {
+            "attachment": {
+              "contentType": "application/pdf",
+              "data": "bHVuZzZtb3d0",
+              "title": "Radiology study observation note"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:c97b7902-58ea-4a8c-92fd-4a40913adc8b",
+      "resource": {
+        "resourceType": "MedicationRequest",
+        "id": "c97b7902-58ea-4a8c-92fd-4a40913adc8b",
+        "meta": {
+          "versionId": "0",
+          "lastUpdated": "2024-08-06T12:18:11.223+05:30",
+          "profile": [
+            "https://nrces.in/ndhm/fhir/r4/StructureDefinition/MedicationRequest"
+          ]
+        },
+        "status": "active",
+        "intent": "plan",
+        "medicationCodeableConcept": {
+          "coding": [
+            {
+              "code": "785686003",
+              "system": "http://snomed.info/sct",
+              "display": "Type of induction"
+            }
+          ],
+          "text": "Type of induction"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "type": "Patient"
+        },
+        "authoredOn": "2024-08-06T12:18:11+05:30",
+        "requester": {
+          "reference": "urn:uuid:41295111-04f9-4b83-b186-ef2975db1c7e",
+          "type": "Practitioner"
+        },
+        "dosageInstruction": [
+          {
+            "doseAndRate": [
+              {
+                "doseQuantity": {
+                  "value": 500,
+                  "code": "mg",
+                  "system": "http://unitsofmeasure.org",
+                  "unit": "mg"
+                },
+                "rateQuantity": {
+                  "value": 3,
+                  "code": "{1/d}",
+                  "system": "http://unitsofmeasure.org",
+                  "unit": "1/d"
+                }
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:84100a2b-c661-4094-8353-3de0356f238b",
+      "resource": {
+        "resourceType": "MedicationRequest",
+        "id": "84100a2b-c661-4094-8353-3de0356f238b",
+        "meta": {
+          "versionId": "0",
+          "lastUpdated": "2024-08-06T12:18:11.223+05:30",
+          "profile": [
+            "https://nrces.in/ndhm/fhir/r4/StructureDefinition/MedicationRequest"
+          ]
+        },
+        "status": "active",
+        "intent": "plan",
+        "medicationCodeableConcept": {
+          "coding": [
+            {
+              "code": "785686003",
```

Comment:
this is same code for Amoxicillin anhydrous. Change to a different medication code please